process/dataPreProcessor.go

```diff
@@ -141,7 +134,7 @@ func (d *eventsDataPreProcessor) RevertIndexedBlock(blockData *outport.BlockData
 	}
 
 	revertData := &data.RevertBlock{
-		Hash:  hex.EncodeToString(header.GetRootHash()),
+		Hash:  hex.EncodeToString(blockData.GetHeaderHash()),
```

Comment:
Why did we use RootHash before, but now we use HeaderHash?
Was this a bug ?

Reply:
yes, it was a bug from previous PR